dags/common/utils.py

```diff
@@ -49,3 +49,10 @@ def check_value(value):
             return all(value)
         return False
     return False
+
+
+def parse_to_int(value):
+    try:
+        return int(value)
+    except (ValueError, TypeError):
+        logger.error("Cannot parse to integer", value=value)
```

Comment:
This logger doesn't accept extra params, I would suggest initialise a new one with structlog and also please update this line with the new one https://github.com/cern-sis/workflows/pull/69/files#diff-caa66940a5e0989c3d3a1a153a6658ead25d65b7f12ccef0d0efb3d46c032cecR30